### DISCUSSION TITLE
Edit item after created

### DISCUSSION THREAD

**Discussion by jtfinley72** [Lissy93/dashy#376] [Help and Support]
Hello.  I'm somewhat new to docker, however Ive deployed the image twice with same result.  If I do not populate all the fields when creating an item, going back to edit later does not save ie the SAVE button doesn't take.  The only way to update after creation is using Update Configuration > Edit Config.  What am I doing wrong here?  I've tried multiple browsers (chrome/safari/firefox). I'm wanting to switch from Homer

**Comment by EVOTk** [comment]
Hello,
I have the same behavior.
You must fill in at least "Item Text", "Service URL" and "Opening Method" for the saving of the item to work correctly.
If the item does not contain at least these 3 filled fields, it is not really saved.

**Reply by jtfinley72** [reply]
Well, your reply made me do something different.  If I edit or change (2) items, it allows me to edit/save.  Odd but it works.  Thank you!

**Reply by lissy93** [reply]
This sounds like a bug, sorry about that, and thanks for raising.
I will look into it, and update you with a fix this weekend :)

**Reply by jtfinley72** [reply]
Well, leave it to me to find'em!  Thanks for response and thank you for your work on this...
